querydsl-tooling/querydsl-ksp-codegen/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.github.openfeign.querydsl</groupId>
+    <artifactId>querydsl-tooling</artifactId>
+    <version>6.9-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>querydsl-ksp-codegen</artifactId>
+  <name>Querydsl - Kotlin KSP codegen support</name>
+
+  <properties>
+    <surefire.useManifestOnlyJar>false</surefire.useManifestOnlyJar>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.jetbrains.kotlin</groupId>
+      <artifactId>kotlin-stdlib</artifactId>
+      <version>${kotlin.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.github.openfeign.querydsl</groupId>
+      <artifactId>querydsl-core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.squareup</groupId>
+      <artifactId>kotlinpoet</artifactId>
+      <version>1.18.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.squareup</groupId>
+      <artifactId>kotlinpoet-ksp</artifactId>
+      <version>1.18.1</version>
```

Comment:
Can we get a property for this version?!

Reply:
ye, i see that kotlin-codegen is using this aswell so i can update their pom for property too